enclave_wrangler/objects_api.py

```diff
@@ -1123,6 +1122,60 @@ def update_cset_metadata_from_objs(
         conn.close()
 
 
+def sync_cset_expression_changes(cset: Dict[str, Any], con: Connection = None):
```

Comment:
## Sync cset expression additions/deletions
This is now being done.

## Updates
I wanted to do updates as well, but there was no time for that.

This means that when we refresh members items, they'll be correct, but the expression definitions might not be if they change between initial fetch and finalization.